cmd/krew/cmd/index.go

```diff
@@ -78,7 +79,7 @@ var indexAddCmd = &cobra.Command{
 		if err != nil {
 			return err
 		}
-		internal.PrintWarning(os.Stderr, `You have added a new index from %q
+		internal.PrintWarning(color.Error, `You have added a new index from %q
```

Comment:
thanks for this patch. the only issue I am seeing is that subbing these with `color.Error` will make it harder to find usages of `os.Stderr` in the future.

e.g. today we have the following occurrences, majority of which are in `cmd/krew/cmd` package. If it makes sense, let's extract these to a package-level variable (`var stderr = color.Error`), and make the method in `cmd/krew/cmd/internal/security_notice.go` accept an io.Writer. what do you think? 

```
cmd/krew/cmd/upgrade.go:97:					fmt.Fprintf(os.Stderr, "Upgrading plugin: %s\n", pluginDisplayName)
cmd/krew/cmd/upgrade.go:100:						fmt.Fprintf(os.Stderr, "Skipping plugin %s, it is already on the newest version\n", pluginDisplayName)
cmd/krew/cmd/upgrade.go:107:						fmt.Fprintf(os.Stderr, "WARNING: failed to upgrade plugin %q, skipping (error: %v)\n", pluginDisplayName, err)
cmd/krew/cmd/upgrade.go:112:				fmt.Fprintf(os.Stderr, "Upgraded plugin: %s\n", pluginDisplayName)
cmd/krew/cmd/upgrade.go:118:				fmt.Fprintf(os.Stderr, "WARNING: Some plugins failed to upgrade, check logs above.\n")
cmd/krew/cmd/index.go:81:		internal.PrintWarning(os.Stderr, `You have added a new index from %q
cmd/krew/cmd/index.go:120:		internal.PrintWarning(os.Stderr, `Plugins [%s] are still installed from index %q!
cmd/krew/cmd/update.go:146:	fmt.Fprintf(os.Stderr, "Adding \"default\" plugin index from %s.\n", defaultIndex)
cmd/krew/cmd/update.go:177:			fmt.Fprintln(os.Stderr, "Updated the local copy of plugin index.")
cmd/krew/cmd/update.go:179:			fmt.Fprintf(os.Stderr, "Updated the local copy of plugin index %q.\n", idx.Name)
cmd/krew/cmd/update.go:194:		showUpdatedPlugins(os.Stderr, preUpdatePlugins, postUpdatePlugins, installedPlugins)
cmd/krew/cmd/uninstall.go:51:			fmt.Fprintf(os.Stderr, "Uninstalled plugin: %s\n", name)
cmd/krew/cmd/internal/security_notice.go:31:	PrintWarning(os.Stderr, securityNoticeFmt+"\n", plugin)
cmd/krew/cmd/root.go:79:			fmt.Fprintln(os.Stderr, err)
cmd/krew/cmd/root.go:114:		internal.PrintWarning(os.Stderr, internal.SetupInstructions()+"\n\n")
cmd/krew/cmd/root.go:146:		fmt.Fprintln(os.Stderr, `This version of Krew is not supported anymore. Please manually migrate:
cmd/krew/cmd/root.go:191:		color.New(color.Bold).Fprintf(os.Stderr, upgradeNotification, version.GitTag(), latestTag)
cmd/krew/cmd/install.go:77:				fmt.Fprintln(os.Stderr, "WARNING: Detected stdin, but discarding it because of --manifest or args")
cmd/krew/cmd/install.go:81:				fmt.Fprintln(os.Stderr, "Reading plugin names via stdin")
cmd/krew/cmd/install.go:155:				fmt.Fprintf(os.Stderr, "Installing plugin: %s\n", plugin.Name)
cmd/krew/cmd/install.go:171:				fmt.Fprintf(os.Stderr, "Installed plugin: %s\n", plugin.Name)
cmd/krew/cmd/install.go:179:				fmt.Fprintln(os.Stderr, indent(output))
internal/gitutil/git.go:85:		w = io.MultiWriter(w, os.Stderr)
```

Reply:
Updated patch. How about this?